db/db.go

```diff
@@ -1173,6 +1173,15 @@ func (db database) AddAndUpdateBudget(budget BudgetStoreData) BudgetHistory {
 	return budgetHistory
 }
 
+func (db database) WithdrawBudget(org_uuid string, amount uint) {
```

Comment:
are we only allowing the user to withdraw their whole budget and no partial withdrawls?

Reply:
It should also include partial withdrawals. But I'm not sure any restriction is required because they're pasting in an invoice.

Reply:
> are we only allowing the user to withdraw their whole budget and no partial withdrawls?

@kevkevinpal They can withdraw any amount of their choice, as long as it's not greater than the total budget they have deposited.